File: 02490.py
for i in range(3):
    cnt = [0, 0]
    a, b, c, d = map(int, input().split())
    cnt[a] += 1
    cnt[b] += 1
    cnt[c] += 1
    cnt[d] += 1
    
    if cnt[0] == 1:
        print('A')
    elif cnt[0] == 2:
        print('B')
    elif cnt[0] == 3:
        print('C')
    elif cnt[0] == 4:
        print('D')
    else:
        print('E')

File: 02490.cpp
#include <iostream>
using namespace std;

int main() {
	for (int i = 0; i < 3; i++) {
		int cnt[2] = { 0,0 };
		int a, b, c, d; cin >> a >> b >> c >> d;
		cnt[a]++; cnt[b]++; cnt[c]++; cnt[d]++;

		if (cnt[0] == 1) cout << "A\n";
		else if (cnt[0] == 2) cout << "B\n";
		else if (cnt[0] == 3) cout << "C\n";
		else if (cnt[0] == 4) cout << "D\n";
		else cout << "E\n";
	}
}
